content/blog/product-roadmap-with-azure-devops.md

```diff
@@ -0,0 +1,112 @@
+## <a name="_4b6ssfq0erd1"></a>Objectives Of A Product Roadmap
+A product roadmap is a tool that requires regular review from all the stakeholders involved. Every feature on the product roadmap is always associated with the defined business goal. Hence it becomes an iterative process. 
+
+The objectives of a product roadmap include—
+
+- Bring stakeholders and team members to an agreement on the direction and priorities of a product 
+- Communicate and track progress toward achieving them
+- Help the product team stay focused on the right goals
+- Ensure stakeholders are aware of the rationale behind decisions
+
+Regular and continuous feedback from end users is incorporated into the product roadmap. Hence it is a living, breathing document that is evolving based on the changing business scenarios and user interactions. 
+
+
+## <a name="_mtx0p0qo5vuo"></a>Revise Product Roadmap Based On Project Goals
+A product roadmap undergoes changes based on three factors—
+
+- the specific product,
+- the stage of its development, and,
+- organization-level choices.
+
+A product in its initial development stage may require a roadmap to be updated monthly or quarterly. Whereas a mature product may only require a roadmap to be updated annually.
+
+Acquisition, retention, and revenue could also be some stages of a product that could define the frequency of the product road mapping. 
+
+Regardless of the stage of development, a product roadmap should be flexible enough to adapt to changes in the market or product itself.
+
+As a rule of thumb, companies should update their product roadmap every quarter or whenever there is a significant change in strategy or market conditions.
+
+## <a name="_d3hs5fxdgurg"></a>Tools to Build Product Roadmaps  
+
+There are many tools available in the market to help configure a product roadmap. Some are Aha, Roadmunk, Productplan,  Asana, Jira, Miro, etc. 
+
+We have been using Azure DevOps for almost 3 years now and this is how we define product roadmaps with Azure DevOps (ADO).
+
+- A default product roadmap feature is available on Azure DevOps. Epics roadmap feature in Azure DevOps is accessed by navigating backlog or boards. 
+ 
+- Azure DevOps roadmap feature is not necessary if a product has individual teams with their own roadmaps.
+
+- Azure DevOps can be configured in a way that stakeholders can create their own roadmaps using queries, tags, and proper definitions of features and associating respective stories.  
+<a name="_412iyj325u33"></a>Step-By-Step Guide On How To Define Product Roadmaps on Azure DevOps
+
+------------------------------------------------------------------------------------------------
+- Create Features with a common tag. 
+  This will help us as an initial filter in queries to get the team’s related features. 
+
+![Screenshot of Azure DevOps Product Roadmap feature](/images/2023/02/product-roadmap.001.png)
```

Comment:
Image loading syntax is incorrect, use this instead:

```
{{< figure src="images/2023/02/product-roadmap.001.png" >}}

```

Same for other screenshots

Reply:
fixed this issue.